frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FieldSettingsPopover.tsx

```diff
@@ -208,16 +216,18 @@ function RequiredInput({
   return (
     <div>
       <ToggleContainer>
-        <strong>{t`Required`}</strong>
+        <RequiredToggleLabel htmlFor="is-required">{t`Required`}</RequiredToggleLabel>
         <Toggle
+          id="is-required"
           value={!!value}
           onChange={required => onChange({ required, defaultValue })}
         />
       </ToggleContainer>
       {!value && (
         <>
-          <SectionLabel>{t`Default Value`}</SectionLabel>
+          <SectionLabel htmlFor="default-value">{t`Default value`}</SectionLabel>
```

Comment:
We prefer not to use title case and [there's a two months old string like that already](https://github.com/metabase/metabase/blob/99c03061388722d7a75c8ecc3756784c6d4a3578/frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx#L84)